package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "seq-logging",
-  "version": "2.0.0",
+  "version": "2.1.0",
   "description": "Sends structured log events to the Seq HTTP ingestion API",
```

Comment:
I would still added `export` property with specific `node` and `browser` properties. In most cases it will allow the consuming developer to import this module with `'seq-logging'` out of the box, without thinking in which env are they working.

Reply:
That sounds useful. How would it help the consuming developer?

Reply:
It works like the `main` property but more specific. You can read in the [node docs](https://nodejs.org/api/packages.html#community-conditions-definitions). Basically, node env will pickup what is specified in `exports.node` and bundlers for browsers will pickup what is in `exports.browser`, `browser` is not official but it is a community standard for a long time already.

Reply:
I like this idea but giving up for now. The node docs define well known keys `node` and `default` but webpack fails if the key `node` is present. 

`Module not found: Error: Exports field key should be relative path and start with "." (key: "node")`